owncloudApp/src/main/res/values/strings.xml

```diff
@@ -730,6 +730,8 @@
     <string name="release_notes_4_3_0_subtitle_accessibility_improvements">Some improvements to make the application more accessible</string>
     <string name="release_notes_4_4_0_title_improved_from_original_folder_auto_upload">Auto uploads more consistent</string>
     <string name="release_notes_4_4_0_subtitle_improved_from_original_folder_auto_upload">Added some mechanisms over retries to improve reliability</string>
+    <string name="release_notes_4_4_0_title_resharing_capability">UX improvement in private shares</string>
```

Comment:
I would change `UX` for another name that would be understood by a normal user.

Reply:
Done